Dockerfile.creator

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.21
+
+ARG TAG
+
+# Copy the binary
+COPY cmd/creator/creator-demo /app/creator-demo
+
+# Copy the configuration folder
+COPY cmd/creator/res /app/res
+
+# Set the environment variable
+ENV TAG=${TAG}
+
+# Set the working directory to /app where both the executable and res folder are located
+WORKDIR /app
+
+ENTRYPOINT ["/app/creator-demo", "-cfg", "./res/config-mqtt.json"]
```

Comment:
@michaelehab Overall this PR looks good but unfortunately this approach stomps on the config files and assumes we'll always want to run the services in a Dockerized context. The various config-mqtt.json files now serve the purpose of allowing someone to run the services natively and use MQTT as the broker. What you'll need to do is create a clone of this file in each app (such as config-docker.json) which uses the appropriate hosts for the Docker environment.